### PR TITLE
Updating workflows/epigenetics/hic-hicup-cooler from 0.1 to 0.2

### PR DESCRIPTION
**Pull request by planemo-autoupdate**
Hello! This is an automated update of the following workflow: **workflows/epigenetics/hic-hicup-cooler**. I created this PR because I think one or more of the component tools are out of date, i.e. there is a newer version available on the ToolShed.

By comparing with the latest versions available on the ToolShed, it seems the following tools are outdated:
* `toolshed.g2.bx.psu.edu/repos/iuc/pygenometracks/pygenomeTracks/3.7+galaxy0` should be updated to `toolshed.g2.bx.psu.edu/repos/iuc/pygenometracks/pygenomeTracks/3.8+galaxy0`
* `toolshed.g2.bx.psu.edu/repos/iuc/pygenometracks/pygenomeTracks/3.7+galaxy0` should be updated to `toolshed.g2.bx.psu.edu/repos/iuc/pygenometracks/pygenomeTracks/3.8+galaxy0`

The workflow release number has been updated from 0.1 to 0.2.

### CODE REVIEW

**Comment by lldelisle** [comment]
Do not merge as I want to upgrade hicup + workflow

File: workflows/epigenetics/hic-hicup-cooler/hic_fastq_to_cool_hicup_cooler.ga [hunk 1]
```diff
@@ -690,9 +690,9 @@
                     }
                 },
                 "tags": "",
-                "uuid": "cc08c539-5d45-4137-95c2-07fb1b8b8a73"
+                "uuid": "54e02e74-a6af-4848-ad37-cc4162a20547"
             },
-            "tool_id": null,
+            "tool_id": 7,
```
Comment (mvdbeek) [inline workflows/epigenetics/hic-hicup-cooler/hic_fastq_to_cool_hicup_cooler.ga:695]: Oops, I think we need to update Galaxy's master branch with fixes from 22.05

Comment (mvdbeek) [reply workflows/epigenetics/hic-hicup-cooler/hic_fastq_to_cool_hicup_cooler.ga:695]: Hmm, the master branch is up to date, this is weird ...

Comment (mvdbeek) [reply workflows/epigenetics/hic-hicup-cooler/hic_fastq_to_cool_hicup_cooler.ga:695]: Will be fixed in https://github.com/galaxyproject/galaxy/pull/15613

**Comment by lldelisle** [comment]
If someone wants to review... If not I will wait for a day and merge.

**Pull request merged**